### DISCUSSION TITLE
Leaderboard isn't shown sometimes in multi

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#16233] [Q&A] [answered]
Happened earlier in karthy's christmas room. I don't know if it's skin related, i've used mine in multi extensively and I never saw it happening before. It may have to do with there being many players in the leaderboard? But then again, on some maps it was present, and on others it wasn't.

![osu_2021-12-23_22-43-03](https://user-images.githubusercontent.com/53081750/147295533-042e0768-e3d0-4975-83db-681e93fd4796.png)

Skin (modified via the in-game skin editor and and exported through the setting in game)
https://drive.google.com/file/d/1D2P635o3KCiU27YDeAN8UGrClUI2qNkn/view?usp=sharing

A beatmap in which this happened
https://osu.ppy.sh/beatmapsets/1324802#osu/2744045

logs
[runtime.log](https://github.com/ppy/osu/files/7771621/runtime.log)
[database.log](https://github.com/ppy/osu/files/7771622/database.log)
[network.log](https://github.com/ppy/osu/files/7771623/network.log)
[performance.log](https://github.com/ppy/osu/files/7771624/performance.log)

**Comment by peppy** [comment]
Hard to tell from the logs, but probably related to user details not being loaded. Were there occasions when it did appear, but took time to do so? Or was it always either there or not there for the full play session?

**Reply by Theighlin** [reply]
Not there for the full map

**Comment by bdach** [comment, marked as answer]
This may have been caused by the same thing that caused beatmap panels to not load, and therefore potentially fixed by https://github.com/ppy/osu/pull/16200.

**Reply by peppy** [reply]
This is the more likely answer, given the above response (that it never recovers).
